mkdocs.yml

```diff
@@ -48,8 +44,10 @@ markdown_extensions:
   - tables
   - toc
   - fenced_code
+  - smarty
   - mdx_math:
       enable_dollar_delimiter: True #for use of inline $..$  
   - footnotes
 
-copyright: Copyright &copy; 2016 <a href="https://github.com/hyperledger/fabric/blob/master/docs/dev-setup/headers.txt">IBM Corp.</a> <br></br>Contributed to the Hyperledger Project under the Apache Software License 2.0
+copyright: Copyright &copy; 2016 <a href="https://github.com/hyperledger/fabric/blob/master/docs/dev-setup/headers.txt" target="_blank">IBM Corp.</a> <br></br> Contributed to the Hyperledger Project under the <a href="https://github.com/hyperledger/fabric/blob/master/LICENSE" target="_blank">Apache Software License 2.0</a>
```

Comment:
The attempt was to link to the Apache license text. If it would be more appropriate to link directly here, that can be achieved: http://www.apache.org/licenses/LICENSE-2.0